src/wrapper.rs

```diff
@@ -200,7 +200,7 @@ where
         buf,
         "fn descriptor(&self) -> &'static ::protobuf::reflect::MessageDescriptor {{ Self::descriptor_static() }}",
     )?;
-    writeln!(buf, "fn new() -> Self {{ Self::new_() }}",)?;
+    writeln!(buf, "fn new() -> Self {{ Self::default() }}",)?;
```

Comment:
I think we should only do this if we are not creating the `new_` ctors (seems weird to generate them and not use them)

Reply:
Then why on earth we're generating them?

I wanted to ask this for long.

Reply:
* I wanted the minimal amount of conversion for library clients - ideally we'd just use `new`
* Prost generates a different `new`, so we can't
* I didn't want to think about it yet, so chose something which is easy to search and replace (`default` is not, because there are other uses) and close to `new`
* The intention is to replace it properly once the first phase of the migration is done.